src/Containers/PictureCard.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { MainInterface, ResponsiveInterface } from '@Utils/BaseStyles';
+import styled, { DefaultTheme } from 'styled-components';
+import { StyledIcon } from 'styled-icons/types';
+import { flex, position } from '@Utils/Mixins';
+import { Card as C } from './Card';
+
+interface TagProps {
+    icon: StyledIcon;
+    text: string;
+}
+
+
+interface PictureCardProps extends MainInterface, ResponsiveInterface {
+    image: string;
+    tags?: TagProps[];
+    theme?: DefaultTheme;
```

Comment:
We dont need this since we dont have withTheme HOC, so remove this and from props